libiop/bcs/merkle_tree.hpp

```diff
@@ -46,6 +46,9 @@ template<typename FieldT, typename hash_digest_type>
 class merkle_tree {
 protected:
     bool constructed_;
+    /* inner_nodes_ is a vector of the (num_leaves - 1) nodes in the tree, with the root at
```

Comment:
Question: looking at the other code (https://github.com/scipr-lab/libiop/blob/5b276ee9607991a404b78655777f9eae572b8499a/libiop/bcs/merkle_tree.tcc#L157), it seems that, in the comment here, this vector is indeed a vector of the (2 * num_leaves - 1) rather than (num_leaves - 1)?